src/consensus/vote.cpp

```diff
@@ -85,6 +85,7 @@ void Vote::voter() const {
   if (cached_voter_) return;
   cached_voter_ = dev::recover(vote_signature_, sha3(false));
```

Comment:
Let's just clean this up little bit. Why to use voter() function at all ? It would be better to do this caching in getVoter()